package/MDAnalysis/transformations/positionaveraging.py

```diff
@@ -0,0 +1,102 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+
+
+"""
+Trajectory Coordinate Averaging --- 
+=================================
+
+Averages the coordinates of a given trajectory with the N previous frames.
+For frames < N, the average of the frames iterated up to that point will be
+returned. 
+
+.. autoclass:: PositionAverager
+
+"""
+from __future__ import absolute_import
+
+import numpy as np
+import warnings
+
+
+
+class PositionAverager:  
+    """
+    
+    Averages the coordinates of a given timestep with the N previous frames. 
+    For frames < N, the average of the frames iterated up to that point will be
+    returned.  
```

Comment:
Explain what happens when
- direction is reversed
- frames are skipped
- arbitrary frames are accessed